.github/workflows/ci-mingw.yml

```diff
@@ -0,0 +1,88 @@
+# This is a copy of ci-unix-shared-local.yml for building shared libraries
+# with an additional build configuration (using installed deps and dav1d).
+
+name: CI MINGW
+on: [push, pull_request]
+
+permissions:
+  contents: read
+
+# Cancel the workflow if a new one is triggered from the same PR, branch, or tag, except on main.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+jobs:
+  build-shared-installed:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        sys:
+          - mingw64
+          - mingw32
+          - ucrt64
+          - clang64
+
+    steps:
+    - uses: msys2/setup-msys2@v2
+      with:
+        update: true
+        install: >-
+          aom
+          cmake
+          dav1d
+          gcc
+          git
+          libyuv
+          ninja
```

Comment:
To install packages for the appropriate environment, it's best to use the "pacboy" command install of "install", and and ":p" to the package name: https://github.com/msys2/setup-msys2#pacboy

This won't work as is (w/o the prefix), as it'll try to install packages in the base MSYS environment, and they don't exit there...

Reply:
Done